addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -7,30 +7,33 @@
         <div class="container">
             <div class="row align-items-center justify-content-between">
                 <!-- Desktop Mode -->
-                <nav aria-label="breadcrumb" class="col d-none d-md-flex">
+                <nav aria-label="breadcrumb" class="col-md-8 d-none d-md-flex">
                     <ol class="breadcrumb bg-transparent mb-0 pl-0 py-0">
                         <li class="breadcrumb-item">
                             <a href="/slides">Home</a>
                         </li>
-                        <li t-att-class="'breadcrumb-item %s' % ('active' if not search_category and not search_tag and not search_slide_type else '')">
+                        <li t-att-class="'breadcrumb-item %s' % ('active' if not search_category and not search_tag and not search_slide_type and not slide else '')">
                             <a t-att-href="'/slides/%s' % slug(channel)"><span t-esc="channel.name"/></a>
                         </li>
-                        <li class="breadcrumb-item active" t-att-aria-current="'page' and search_category" t-if="search_category">
+                        <li t-att-class="'breadcrumb-item %s' % ('active' if not slide else '')" t-att-aria-current="'page' and search_category" t-if="search_category">
```

Comment:
You use that a lot.
Could be set at the beginning with a <t-set>

`<t t-set="breadcrumb_class" t-value="'breadcrumb-item %s' % ('active' if not slide else '')" />`

And then `t-att-class="breadcrumb_class"`